packages/renderer/src/lib/image/PullImage.spec.ts

```diff
@@ -362,3 +362,47 @@ test('Expect latest tag warning is not displayed when the image has latest tag',
   const errorMesssage = screen.queryByRole('alert', { name: 'Warning Message Content' });
   expect(errorMesssage).toBeNull();
 });
+
+test('Expect isValidName to be truthy === error to be falsy', async () => {
```

Comment:
Could we avoid `===` in test name ? 

```suggestion
test('input component should not raise an error when the input is valid', async () => {
```